openslides/core/static/js/core/projector.js

```diff
@@ -56,6 +56,30 @@ angular.module('OpenSlidesApp.core.projector', ['OpenSlidesApp.core'])
     }
 ])
 
+.controller('LanguageCtrl', [
+    '$scope',
+    'gettextCatalog',
+    'Languages',
+    'Config',
+    'ProjectorID',
+    function ($scope, gettextCatalog, Languages, Config, ProjectorID) {
+        // for the dynamic title
+        $scope.projectorId = ProjectorID();
```

Comment:
No, see Line 99. The container needs to know the projector id for receiving the resolution.